packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue

```diff
@@ -1,11 +1,13 @@
 <template>
 	<div class="policy-group">
 		<div class="form-header">
-			<label class="mr-auto" tag="h5"> {{ config.intervention?.name ?? `Intervention` }}</label>
-			<aside>
-				<label for="active">Optimize</label>
-				<InputSwitch v-model="knobs.isActive" :disabled="true" @change="emit('update-self', knobs)" />
-			</aside>
+			<h6 class="mr-auto">{{ config.intervention?.name ?? `Intervention` }}</h6>
+			<tera-signal-bars
+				v-if="!!knobs.relativeImportance"
+				v-model="knobs.relativeImportance"
+				@update:model-value="emit('update-self', knobs)"
+				label="Relative importance"
+			/>
```

Comment:
What is displayed if there is not `knobs.relativeImportance`? Otherwise, let's move the `v-if` to the parent div.

Reply:
Just the title would be shown.  Currently optimizing dynamic interventions aren't supported so the signal bars here is always hidden (it is always 0).  But its there in case we add support for them.
    
![Screenshot 2024-10-01 at 11 49 53 AM](https://github.com/user-attachments/assets/6ceca39f-0a31-4c45-abfa-1bcf6c77aca0)